### PR TITLE
Quips update, cleaning up and adding some new variants.

### PR DESCRIPTION
**Pull request by ShinodaChan**
Open for suggestions since this should be a pretty fun PR.

### CODE REVIEW

File: Monika After Story/game/script-affection.rpy [hunk 1]
```diff
@@ -946,78 +946,99 @@ init 15 python in mas_affection:
...
+            _("Huh?"),
+            _("Yeah?"),
             _("What do you want?"),
             _("What now?"),
             _("What is it?"),
-#            "Fine...we can talk.",
-#            "Just...whatever, go ahead."
+            _("Go on then."),
+            _("Let's hear it."),
+            _("Is this really what you want?"),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:975]: Not sure how much sense this one makes
```suggestion
```

File: Monika After Story/game/script-affection.rpy [hunk 2]
```diff
@@ -946,78 +946,99 @@ init 15 python in mas_affection:
...
+            _("Don't make fun of me."),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:976]: We should note that *Upset* isn't exactly the best description for it. It's more of a 'bored' or uninterested stage. Kinda like the start of things falling apart. As such, Monika isn't exactly angry at this stage. So this sort of response doesn't full work
```suggestion
```

File: Monika After Story/game/script-affection.rpy [hunk 3]
```diff
@@ -946,78 +946,99 @@ init 15 python in mas_affection:
...
+            _("This better be important."),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:977]: To match with the above, we can slightly modify this to be a little less aggressive
```suggestion
            _("I hope this is important."),
```

File: Monika After Story/game/script-affection.rpy [hunk 4]
```diff
@@ -1042,46 +1063,52 @@ init 15 python in mas_affection:
 
         ## DISTRESSED quips
         quips = [
-            _("...Sure."),
-            _("...Fine."),
-            _("I guess we can play a game."),
-            _("I guess, if you really want to."),
+            _("..."),
+            _("...alright."),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:1067]: consistency
```suggestion
            _("...Alright."),
```

File: Monika After Story/game/script-affection.rpy [hunk 5]
```diff
@@ -1042,46 +1063,52 @@ init 15 python in mas_affection:
...
+            _("...okay."),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:1068]:
```suggestion
            _("...Okay."),
```

File: Monika After Story/game/script-affection.rpy [hunk 6]
```diff
@@ -1042,46 +1063,52 @@ init 15 python in mas_affection:
...
-            _("...Which game?"),
-            _("Okay."),
-            _("Sure."),
-#            "Okay...whatever, choose a game.",
-#            "Fine, pick a game."
+            _("..."),
+            _("...Really?"),
+            _("...Fine."),
+            _("Whatever."),
+            _("This isn't the time to be having fun..."),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:1082]:
```suggestion
```

File: Monika After Story/game/script-affection.rpy [hunk 7]
```diff
@@ -946,78 +946,99 @@ init 15 python in mas_affection:
...
             _("...Yes?"),
-            _("...Oh?"),
-            _("...Huh?"),
             _("...Hm?"),
-            _("We can try talking, I guess."),
-            _("I guess we can talk."),
-            _("Oh... You want to talk?"),
-            _("If you want to talk, go ahead."),
-            _("We can talk if you really want to."),
+            _("...Hello?"),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:952]: I'm not sure how much this works, these bits aren't greetings, and she'll have already greeted you having launched MAS.
```suggestion
```

File: Monika After Story/game/script-affection.rpy [hunk 8]
```diff
@@ -946,78 +946,99 @@ init 15 python in mas_affection:
...
+            _("Oh..."),
+            _("Huh..."),
+            _("...we can talk."),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:955]:
```suggestion
            _("...We can talk."),
```

File: Monika After Story/game/script-affection.rpy [hunk 9]
```diff
@@ -946,78 +946,99 @@ init 15 python in mas_affection:
...
+            _("...go ahead."),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:956]:
```suggestion
            _("...Go ahead."),
```

File: Monika After Story/game/script-affection.rpy [hunk 10]
```diff
@@ -946,78 +946,99 @@ init 15 python in mas_affection:
...
+            _("...okay."),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:957]:
```suggestion
            _("...Okay."),
```

**Review by multimokia — CHANGES_REQUESTED** [review]
Some things

File: Monika After Story/game/script-affection.rpy [hunk 11]
```diff
@@ -946,78 +946,99 @@ init 15 python in mas_affection:
...
+            _("..."),
             _("...Yes?"),
-            _("...Oh?"),
-            _("...Huh?"),
             _("...Hm?"),
-            _("We can try talking, I guess."),
-            _("I guess we can talk."),
-            _("Oh... You want to talk?"),
-            _("If you want to talk, go ahead."),
-            _("We can talk if you really want to."),
```
Comment (fromulus) [inline Monika After Story/game/script-affection.rpy:957]: Not sure why you would want to remove these, I feel they work just fine in that context.
Maybe move them to Upset instead I guess, aside from `"Oh... You want to talk?"` which probably still fits better here.

File: Monika After Story/game/script-affection.rpy [hunk 12]
```diff
@@ -946,78 +946,115 @@ init 15 python in mas_affection:
...
-            _("Anything you'd like to talk about?"),
-            _("We can talk about anything you like, [player].")
+            _("What's up, [mas_get_player_nickname()]?"),
+            _("Ask away, [mas_get_player_nickname()]~"),
+            _("I'm all ears, [mas_get_player_nickname()]~"),
+            _("We can talk about whatever you'd like, [mas_get_player_nickname()]."),
+            _("Of course we can talk, [mas_get_player_nickname()~."),
+            _("Take all the time you need, [mas_get_player_nickname()]."),
+            _("I'm all yours, [mas_get_player_nickname()]~."),
+            _("Oh? Something...{i}important{/i} on your mind, [mas_get_player_nickname()]~."),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:1056]:
```suggestion
            _("Oh? Something...{i}important{/i} on your mind, [mas_get_player_nickname()]~"),
```

File: Monika After Story/game/script-affection.rpy [hunk 13]
```diff
@@ -946,78 +946,115 @@ init 15 python in mas_affection:
...
+            _("Is there something you'd like to talk about, [mas_get_player_nickname()]?"),
+            _("Something on your mind?"),
+            _("Something on your mind, [mas_get_player_nickname()]?"),
+            _("Up to chat, I see~."),
             _("Yes, [mas_get_player_nickname()]?"),
-            _("What's on your mind, [mas_get_player_nickname()]?"),
+            _("What's up, [player]?"),
+            _("Ask away, [mas_get_player_nickname()]~"),
+            _("I'm all ears, [mas_get_player_nickname()]~"),
+            _("Of course we can talk, [mas_get_player_nickname()~."),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:1033]:
```suggestion
            _("Of course we can talk, [mas_get_player_nickname()~"),
```

File: Monika After Story/game/script-affection.rpy [hunk 14]
```diff
@@ -1042,54 +1079,74 @@ init 15 python in mas_affection:
...
 
         ## ENAMORED quips
         quips = [
             _("What would you like to play? <3"),
+            _("Pick a game, any game~."),
             _("Choose anything you like, [mas_get_player_nickname()]."),
             _("Pick anything you like, [mas_get_player_nickname()]."),
+            _("I hope you're ready, [mas_get_player_nickname()]~."),
+            _("Let's have some fun, [mas_get_player_nickname()]."),
+            _("Don't expect me to go easy on you, [mas_get_player_nickname()]~!"),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:1145]:
```suggestion
            _("Don't expect me to go easy on you, [mas_get_player_nickname()]!~"),
```

File: Monika After Story/game/script-affection.rpy [hunk 15]
```diff
@@ -1042,54 +1079,74 @@ init 15 python in mas_affection:
...
         ]
         save_quips(AFFECTIONATE, quips)
 
         ## ENAMORED quips
         quips = [
             _("What would you like to play? <3"),
+            _("Pick a game, any game~."),
             _("Choose anything you like, [mas_get_player_nickname()]."),
             _("Pick anything you like, [mas_get_player_nickname()]."),
+            _("I hope you're ready, [mas_get_player_nickname()]~."),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:1143]:
```suggestion
            _("I hope you're ready, [mas_get_player_nickname()]~"),
```

File: Monika After Story/game/script-affection.rpy [hunk 16]
```diff
@@ -1042,54 +1079,74 @@ init 15 python in mas_affection:
...
+            _("Let's have some fun, [mas_get_player_nickname()]."),
+            _("Don't expect me to go easy on you, [mas_get_player_nickname()]~!"),
+            _("Let's do this, [mas_get_player_nickname()]~!"),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:1146]:
```suggestion
            _("Let's do this, [mas_get_player_nickname()]!~"),
```

File: Monika After Story/game/script-affection.rpy [hunk 17]
```diff
@@ -1098,11 +1155,21 @@ init 15 python in mas_affection:
             _("What would you like to play? <3"),
             _("Choose anything you like, [mas_get_player_nickname()]."),
             _("Pick anything you like, [mas_get_player_nickname()]."),
-            _("Yay! Let's play together!"),
-            _("I'd love to play something with you!"),
-            _("I'd love to play with you!")
+            _("Let's do this, [mas_get_player_nickname()]!~"),
+            _("Bring it on, [mas_get_player_nickname()]."),
+            _("Ready to lose, [mas_get_player_nickname()]?"),
+            _("I hope you're ready, [mas_get_player_nickname()]~."),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:1161]:
```suggestion
            _("I hope you're ready, [mas_get_player_nickname()]~"),
```

File: Monika After Story/game/script-affection.rpy [hunk 18]
```diff
@@ -1098,11 +1155,21 @@ init 15 python in mas_affection:
...
+            _("Let's have some fun, [mas_get_player_nickname()]."),
+            _("Don't expect me to go easy on you, [mas_get_player_nickname()]~!"),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:1163]:
```suggestion
            _("Don't expect me to go easy on you, [mas_get_player_nickname()]!~"),
```

File: Monika After Story/game/script-affection.rpy [hunk 19]
```diff
@@ -1098,11 +1155,21 @@ init 15 python in mas_affection:
...
+            _("Pick a game, any game~"),
+            _("Throwing down the gauntlet, are we?"),
+            _("It's time to duel!"),
+            _("Challenge accepted!"),
+            _("I'm ready when you are, [mas_get_player_nickname()]!"),
+            _("I'd love to play something with you, [mas_get_player_nickname()]~."),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:1169]:
```suggestion
            _("I'd love to play something with you, [mas_get_player_nickname()]~"),
```

File: Monika After Story/game/script-affection.rpy [hunk 20]
```diff
@@ -1098,11 +1155,21 @@ init 15 python in mas_affection:
...
+            _("Sure, I'd love to play with you!"),
+            _("I'll always be up to play with you, [mas_get_player_nickname()]~."),
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy:1171]:
```suggestion
            _("I'll always be up to play with you, [mas_get_player_nickname()]~"),
```

**Review by multimokia — CHANGES_REQUESTED** [review]
Consistency

File: Monika After Story/game/script-affection.rpy [hunk 21]
```diff
@@ -946,78 +946,115 @@ init 15 python in mas_affection:
...
             _("<3"),
-            _("Anything you'd like to talk about?"),
-            _("We can talk about anything you like, [player].")
+            _("What's up, [mas_get_player_nickname()]?"),
+            _("Ask away, [mas_get_player_nickname()].~"),
+            _("I'm all ears, [mas_get_player_nickname()].~"),
+            _("We can talk about whatever you'd like, [mas_get_player_nickname()]."),
+            _("Of course we can talk, [mas_get_player_nickname().~"),
+            _("Take all the time you need, [mas_get_player_nickname()]."),
+            _("I'm all yours, [mas_get_player_nickname()].~"),
```
Comment (Booplicate) [inline Monika After Story/game/script-affection.rpy:1055]: We don't use `.` and `~` at once.

File: Monika After Story/game/script-affection.rpy [hunk 22]
```diff
@@ -1042,54 +1079,74 @@ init 15 python in mas_affection:
...
         ]
         save_quips(AFFECTIONATE, quips)
 
         ## ENAMORED quips
         quips = [
             _("What would you like to play? <3"),
+            _("Pick a game, any game~."),
             _("Choose anything you like, [mas_get_player_nickname()]."),
             _("Pick anything you like, [mas_get_player_nickname()]."),
+            _("I hope you're ready, [mas_get_player_nickname()].~"),
```
Comment (Booplicate) [inline Monika After Story/game/script-affection.rpy:1143]: We don't use `.` and `~` at once.

File: Monika After Story/game/script-affection.rpy [hunk 23]
```diff
@@ -1098,11 +1155,21 @@ init 15 python in mas_affection:
             _("What would you like to play? <3"),
             _("Choose anything you like, [mas_get_player_nickname()]."),
             _("Pick anything you like, [mas_get_player_nickname()]."),
-            _("Yay! Let's play together!"),
-            _("I'd love to play something with you!"),
-            _("I'd love to play with you!")
+            _("Let's do this, [mas_get_player_nickname()]!~"),
+            _("Bring it on, [mas_get_player_nickname()]."),
+            _("Ready to lose, [mas_get_player_nickname()]?~"),
+            _("I hope you're ready, [mas_get_player_nickname()].~"),
```
Comment (Booplicate) [inline Monika After Story/game/script-affection.rpy:1161]: We don't use `.` and `~` at once.

File: Monika After Story/game/script-affection.rpy [hunk 24]
```diff
@@ -1098,11 +1155,21 @@ init 15 python in mas_affection:
...
+            _("Let's have some fun, [mas_get_player_nickname()]."),
+            _("Don't expect me to go easy on you, [mas_get_player_nickname()]!~"),
+            _("Pick a game, any game.~"),
+            _("Throwing down the gauntlet, are we?"),
+            _("It's time to duel!"),
+            _("Challenge accepted!"),
+            _("I'm ready when you are, [mas_get_player_nickname()]!"),
+            _("I'd love to play something with you, [mas_get_player_nickname()].~"),
```
Comment (Booplicate) [inline Monika After Story/game/script-affection.rpy:1169]: We don't use `.` and `~` at once.

File: Monika After Story/game/script-affection.rpy [hunk 25]
```diff
@@ -1098,11 +1155,21 @@ init 15 python in mas_affection:
...
+            _("Sure, I'd love to play with you!"),
+            _("I'll always be up to play with you, [mas_get_player_nickname()].~"),
```
Comment (Booplicate) [inline Monika After Story/game/script-affection.rpy:1171]: We don't use `.` and `~` at once.

**Review by Booplicate — CHANGES_REQUESTED** [review]
Some more bits.

File: Monika After Story/game/script-affection.rpy [hunk 26]
```diff
@@ -946,78 +946,115 @@ init 15 python in mas_affection:
...
-            _("What would you like to talk about? <3"),
+            _("What would you like to talk about?"),
             _("What would you like to talk about, [mas_get_player_nickname()]?"),
+            _("What are you thinking of?"),
+            _("Is there something you'd like to talk about, [mas_get_player_nickname()]?"),
+            _("Something on your mind?"),
+            _("Something on your mind, [mas_get_player_nickname()]?"),
+            _("Up to chat, [player]?"),
             _("Yes, [mas_get_player_nickname()]?"),
-            _("What's on your mind, [mas_get_player_nickname()]?"),
```
[the remark this answers is not in the document]
Comment (ShinodaChan) [reply Monika After Story/game/script-affection.rpy:994]: Very similar to "Something on your mind" IMO

[the remark this answers is not in the document]
Comment (Booplicate) [reply Monika After Story/game/script-affection.rpy:994]: But that's good. Similar lines said differently are good for quips.

[the remark this answers is not in the document]
Comment (jmwall24) [reply Monika After Story/game/script-affection.rpy:994]: That's fine though. Having variants is a good thing

**Review by jmwall24 — CHANGES_REQUESTED** [review]

File: Monika After Story/game/script-affection.rpy [hunk 27]
```diff
@@ -946,78 +946,119 @@ init 15 python in mas_affection:
...
-            _("...Hm?"),
-            _("We can try talking, I guess."),
+            _("..."),
+            _("Yes?"),
+            _("Oh..."),
+            _("Huh..."),
             _("I guess we can talk."),
-            _("Oh... You want to talk?"),
-            _("If you want to talk, go ahead."),
-            _("We can talk if you really want to."),
```
Comment (jmwall24) [inline Monika After Story/game/script-affection.rpy:957]: Any reason we're deleting these?

**Review by multimokia — APPROVED** [review]

**Review by multimokia — APPROVED** [review]

**Review by jmwall24 — DISMISSED** [review]

**Review by jmwall24 — APPROVED** [review]

**Review by Booplicate — APPROVED** [review]

**Review by multimokia — APPROVED** [review]

**Pull request merged**